src/core/api/index.js

```diff
@@ -34,12 +34,24 @@ export function callApi({ endpoint, schema, params = {}, auth = false, state = {
       options.headers.authorization = `Bearer ${state.token}`;
     }
   }
-  return fetch(`${API_BASE}/${endpoint}/${queryString}`, options)
+  const apiURL = `${API_BASE}/${endpoint}/${queryString}`;
+
+  return fetch(apiURL, options)
     .then((response) => {
+      // FIXME: what about 3xx responses?
```

Comment:
3xx seems to be followed by default. Would we need to do anything other than handling the last response?